core/src/mindustry/core/NetServer.java

```diff
@@ -367,6 +367,10 @@ public VoteSession(VoteSession[] map, Player target){
             }
 
             void vote(Player player, int d){
+                if(d==-2){
```

Comment:
Incorrect formatting (missing spaces around `==`), and this should probably `return` so it doesn't actually do any voting.